src/main/resources/application.conf

```diff
@@ -485,6 +485,28 @@ metronome {
     kill_chunk_size = ${?METRONOME_KILLTASK_KILL_CHUNKSIZE}
     kill_retry_timeout = ${?METRONOME_KILLTASK_KILL_RETRY_TIMEOUT}
   }
+  metrics {
+    prefix = ${?METRONOME_METRIC_PREFIX}
+
+    histogram.reservoir.significant.digits =  ${?METRONOME_HISTOGRAM_RESERVIOR_SIGNIFICANT_DIGITS}
+    histogram.reservoir.reset.periodically =  ${?METRONOME_HISTOGRAM_RESERVIOR_RESET_PERIODICALLY}
+    histogram.reservoir.reset.internal.ms =  ${?METRONOME_HISTOGRAM_RESERVIOR_RESET_INTERNAL_MS}
+    histogram.reservoir.reset.chucks =  ${?METRONOME_HISTOGRAM_RESERVIOR_RESET_CHUCKS}
+
+    statsd.host =  ${?METRONOME_STATSD_HOST}
+    statsd.port =  ${?METRONOME_STATSD_PORT}
+    statsd.transmission.interval.ms =  ${?METRONOME_STATSD_TRANSMISSION_INTERVAL_MS}
+
+    datadog.protocol =  ${?METRONOME_DATADOG_PROTOCOL}
+    datadog.host =  ${?METRONOME_DATADOG_HOST}
+    datadog.port =  ${?METRONOME_DATADOG_PORT}
+    datadog.api.key =  ${?METRONOME_DATADOG_API_KEY}
+    datadog.transmission.interval.ms =  ${?METRONOME_DATADOG_TRANSMISSION_INTERVAL_MS}
+
+    reporter.prometheus = ${?METRONOME_PROMETHEUS_ENABLED}
```

Comment:
we don't have API endpoint for prometheus so I think we don't want this option

Reply:
prometheus polls for metrics... I thought we set it against the `/v1/metrics` end pt...   I was expose the prometheus flag here but I don't understand it and will remove it.

and... I would rather add things when they are present... I will remove all the reporters at this point until we have support for them.